CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+The Arcane Artistry Changelog. We use Epoch-Based Versioning where:
+- `X` is for major releases
+- `Y` is for new features
+- `Z` is for bug fixes, repository changes and code maintenance
```

Comment:
repository changes and code maintenance?
I think its weird to update the version number to include changes that dont affect the mod.jar at all.
Users would suggest something got fixed with the mod which can be confusing I would say.
So, do we only update "Z" if a bug is fixed or for example a texture got updated (without changing file name)?